operations/services.md

```diff
@@ -610,7 +610,9 @@ Returns all restrictions of the default service provided by the enterprise.
 | `Client` | string | required | Name and version of the client application. |
 | `ServiceIds` | array of string | required | Unique identifiers of the [Service](#service)s from which the restrictions are requested. |
 | `ResourceCategoryIds` | array of string | optional | Unique identifiers of [Resource categories](enterprises.md#resource-category). |
-| `RateIds` | array of string | optional | Unique identifiers of [Rate](services.md#rate)s. |
+| `RateIds` | array of string | optional | Unique identifiers of [Rate](#rate)s. Returns all restrictions that affect given rates (i.e. ones without any [Conditions](#restriction-conditions), ones from same [Rate group](services.md#rate-group) as specified rate, or ones inherited from base rate).  |
```

Comment:
Are you sure about same rate group? If it behaves like this, it is really counterintuitive and we should I guess rather fix this.